x-pack/plugins/cases/public/components/create/custom_fields.tsx

```diff
@@ -41,6 +45,17 @@ const CustomFieldsComponent: React.FC<Props> = ({ isLoading, customFieldsConfigu
 
   return (
     <EuiFlexGroup direction="column" gutterSize="s">
+      {customFieldsComponents.length ? (
```

Comment:
It seems to me that the whole component should not be returned if the are no custom fields configured. Maybe we can either return `null` for the whole component (`{customFieldsComponents.length ? ( <EuiFlexGroup direction="column" gutterSize="s">....`) or in `x-pack/plugins/cases/public/components/create/form.tsx` check if `customFieldsConfiguration` and not render `CustomFields` if they are empty.